monitorrent/engine.py

```diff
@@ -588,16 +595,19 @@ def _execute(self, ids=None):
         caught_exception = None
         self.is_executing = True
         try:
+            log.info("Starting execute", time=str(datetime.now()))
             self.logger.started(datetime.now(pytz.utc))
             engine = Engine(self.logger, self.settings_manager, self.trackers_manager,
                             self.clients_manager, self.notifier_manager)
             engine.execute(ids)
         except:
             caught_exception = sys.exc_info()[0]
+            log.error("An error has occurred during execute", exception=str(caught_exception))
```

Comment:
Well, it doesn't, as `sys.exc_info()[0]` returns exception type only (ref to https://docs.python.org/2/library/sys.html#sys.exc_info)
Also, it's worth to mention, that `structlog` has it's own event and render for handling exceptions (take a look at http://structlog.readthedocs.io/en/stable/api.html#structlog.stdlib.BoundLogger.exception)

Btw, may I ask (as not that experienced python developer) - why `sys.exec_info` instead of `traceback.print_exception` ? It's not that safe to assign it to a local variable (from docs and SO discussions).